src/util/DeployCommand.ts

```diff
@@ -50,7 +50,7 @@ export class DeployCommand extends SystemCommand {
                     this.displayError('Please specify \'deploy-image-target\' in cli-config.yml for Kubernetes deployment.');
                 });
             } else {
-                const regex = new RegExp(/registry.*.bluemix.net/);
+                const regex = new RegExp(/registry.*.icr.io/);
```

Comment:
I don't believe the registry begins with `registry` anymore. But just <region>.icr.io.

Reply:
I see. I found some inconsistency in this doc [page](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#deploy) in production.  I can file an issue  with the doc team

Reply:
Yes, please do